x/blackbox/blackbox_test.py

```diff
@@ -0,0 +1,476 @@
+from collections import Counter
```

Comment:
@tzaffi Friendly pointer to unused import.  Perhaps one day _soon_, a linter will help us here.  😅 

```suggestion

```

Reply:
good catch